openrtb_ext/imp_rubicon.go

```diff
@@ -6,9 +6,9 @@ import (
 
 // ExtImpRubicon defines the contract for bidrequest.imp[i].ext.rubicon
 type ExtImpRubicon struct {
-	AccountId int                `json:"accountId"`
-	SiteId    int                `json:"siteId"`
-	ZoneId    int                `json:"zoneId"`
+	AccountId json.RawMessage    `json:"accountId"`
+	SiteId    json.RawMessage    `json:"siteId"`
+	ZoneId    json.RawMessage    `json:"zoneId"`
```

Comment:
Would it be easier to use `json.Number` instead? Please refer to this PR: https://github.com/prebid/prebid-server/pull/2105/files

Reply:
Yeah, thanks for suggestion!

Reply:
Thank you for changing the code! 
May I ask you to add json test where imp.ext.{accountId/siteId/zoneId} have string values? Similar to what you already have in `exemplary`, but with strings instead?